src/coredump/meson.build

```diff
@@ -4,6 +4,8 @@ systemd_coredump_sources = files('''
         coredump.c
         coredump-vacuum.c
         coredump-vacuum.h
+        coredump-ratelimit.c
+        coredump-ratelimit.h
```

Comment:
please keep sorted

Reply:
I have addressed the review comment. Please check once